cmd/thanos/compact.go

```diff
@@ -271,9 +271,11 @@ func runCompact(
 				labelShardedMetaFilter,
 				consistencyDelayMetaFilter,
 				ignoreDeletionMarkFilter,
+				block.NewReplicaLabelRemover(logger, conf.dedupReplicaLabels),
 				duplicateBlocksFilter,
 				noCompactMarkerFilter,
-			}, []block.MetadataModifier{block.NewReplicaLabelRemover(logger, conf.dedupReplicaLabels)},
+				timePartitionMetaFilter,
```

Comment:
`timePartitionMetaFilter` has been listed twice in this list? :thinking:

Reply:
Oh, yeah, I think I messed that up in resolving merge conflicts. Fixed.